app/views/navigation/_mobile.html.haml

```diff
@@ -3,9 +3,10 @@
   .container
     %ul.nav.navbar-nav.navbar-icon-row
       = nav_entry t("navigation.timeline"), root_path, icon: 'home', icon_only: true
-      = nav_entry t("navigation.inbox"), '/inbox',
-        badge: inbox_count, badge_color: 'primary', badge_pill: true,
-        icon: 'inbox', icon_only: true
+      #nav-inbox-mobile
```

Comment:
Please don't write invalid HTML. (first level element inside `ul/ol` needs to be `li` **always**)